### PR TITLE
Added logging-fstring-interpolation and logging-format-interpolation message examples'

### PR DESCRIPTION
**Pull request by matusvalo**
- Added logging-format-interpolation message example
- Added logging-fstring-interpolation message example

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|     | :bug: Bug fix          |
|     | :sparkles: New feature |
|     | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

Related #5953

### CODE REVIEW

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
I think we needs a rationale about the performance implication and why this check exists in the first place (btw, the performance gain are smaller or maybe negligible since f-string were introduced so I don't know what the conclusion should be, I'm personally disabling this check now).

File: doc/data/messages/m/logging-format-interpolation/related.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+- `logging variable data <https://docs.python.org/3/howto/logging.html#logging-variable-data>`_
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/m/logging-format-interpolation/related.rst:1]:
```suggestion
- `logging variable data <https://docs.python.org/3/howto/logging.html#logging-variable-data>`_
- `rational <https://stackoverflow.com/questions/34619790>`_
```

File: doc/data/messages/m/logging-fstring-interpolation/related.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+- `logging variable data <https://docs.python.org/3/howto/logging.html#logging-variable-data>`_
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/m/logging-fstring-interpolation/related.rst:1]:
```suggestion
- `logging variable data <https://docs.python.org/3/howto/logging.html#logging-variable-data>`_
- `rational <https://stackoverflow.com/questions/34619790>`_
```

**Review by Pierre-Sassoulas — APPROVED** [review]

**Comment by matusvalo** [comment]
Thnank you @Pierre-Sassoulas .

**Pull request merged**